src/featurizations/graphnodefeaturization.jl

```diff
@@ -2,7 +2,7 @@ export GraphNodeFeaturization
 export encodable_elements, decode, chunk_vec
 
 """
-    GraphNodeFeaturization(atom_features::Vector{AtomFeatureDescriptor})
+    GraphNodeFeaturization(encoded_atom_features::Vector{AtomFeatureDescriptor})
```

Comment:
both of the instances in this file should just be `atom_features` or `atom_feature_descriptors` or something, they're not encoded features.

Reply:
Initially, I hadn't renamed it. But then, the tests broke, since this is being done in `featurize!()` - 
https://github.com/Chemellia/ChemistryFeaturization.jl/blob/7c8a49562b4372788c2866522f1f8f38f888bcd5/src/ChemistryFeaturization.jl#L136
So, I changed the name over here too.
Renaming it to `atom_feature_descriptors` also would also require redefining `featurize!()`, which is a good thing as it could help reduce the coupling between `AbstractAtoms` and `AbstractFeaturization` objects.

Reply:
ohhh right haha forgot I had done that...but yeah, probably we just need to add a function argument for the corresponding field or something and maybe just ditch the generic implementation